modules/s3-remote-state/README.md

```diff
@@ -37,43 +47,80 @@ output "bucket_name" {
 ```
 
-Review the changes Terraform will make with `tf plan`, and apply those changes
-with `tf apply`.
+Initially the project with `terraform init.` Review the changes Terraform will make with `tf plan --out=tf.out`, and apply those changes with `tf apply tf.out`.
+
+### Initialize the Remote State S3 Backend
 
-Configure remote state to use that bucket with:
+We want to use the newly created S3 bucket to store the Terraform state for this module. There are several steps that need to be taken. Make sure to backup your existing `terraform.tfstate` file before taking these steps:
 
+1. First we have to [initialize the backend][1] and copy over the current state of the backend bucket you just created. Uncomment the following, so that the code in the `main.tf` file looks similar to this:
+```
+terraform {
+  backend "s3" {
+# the other comments remain    
+  }
+}
 ```
-ᐅ tf remote config -backend=s3 -backend-config="bucket=$(tfo bucket_name)" -backend-config="key=$(tfo bucket_name)" -backend-config="encrypt=true" -pull=false
-Remote state management enabled
+2. We are doing what Terraform calls _partial configuration_.  If you now run the command `terraform init` you will get dialog similar to what you see in what follows. Where you are prompted with ** Enter a value:**, you should enter something similar to what you see in the sample. Please note, as described in the [Terraform documentation][1], you can alternatively pass the configuration to initialization using the `cli` , or uncomment all the code in step 1 (the file already includes these parameters):
 ```
+Initializing modules...
+- module.s3-remote-state-bucket
 
-Push the existing state to that bucket:
+Initializing the backend...
+bucket
+  The name of the S3 bucket
 
-```
-ᐅ tf remote push
-State successfully pushed!
-```
+  Enter a value: fitstation-tf-remote-state-test  
+
+key
+  The path to the state file inside the bucket
+
+  Enter a value: mod/s3-remote/terraform.tfstate
 
+Do you want to copy existing state to the new backend?
+  Pre-existing state was found while migrating the previous "local" backend to the
+  newly configured "s3" backend. No existing state was found in the newly
+  configured "s3" backend. Do you want to copy this state to the new "s3"
+  backend? Enter "yes" to copy and "no" to start with an empty state.
+
+  Enter a value: yes
+
+
+Successfully configured the backend "s3"! Terraform will automatically
+use this backend unless the backend configuration changes.
+
+Initializing provider plugins...
+
+The following providers do not have any version constraints in configuration,
+so the latest version was installed.
+
+To prevent automatic upgrades to new major versions that may contain breaking
+changes, it is recommended to add version = "..." constraints to the
+corresponding provider blocks in configuration, with the constraint strings
+suggested below.
+
+* provider.aws: version = "~> 1.14"
+
+Terraform has been successfully initialized!
+```
```

Comment:
add an empty line between the closing quotes and the next line of text.

Reply:
fixed